flow/connectors/utils/postgres.go

```diff
@@ -20,3 +23,30 @@ func GetPGConnectionString(pgConfig *protos.PostgresConfig) string {
 	)
 	return connString
 }
+
+func GetCustomDataType(ctx context.Context, connStr string, dataType uint32) (qvalue.QValueKind, error) {
+	var typeName string
+	pool, err := pgxpool.New(ctx, connStr)
+	if err != nil {
+		return qvalue.QValueKindString, fmt.Errorf("failed to create postgres connection pool"+
+			" for custom type handling: %w", err)
+	}
+
+	defer pool.Close()
+
+	err = pool.QueryRow(ctx, "SELECT typname FROM pg_type WHERE oid = $1", dataType).Scan(&typeName)
+	if err != nil {
+		return qvalue.QValueKindString, fmt.Errorf("failed to query pg_type for custom type handling: %w", err)
+	}
+
+	var qValueKind qvalue.QValueKind
+	switch typeName {
+	case "geometry":
```

Comment:
"point"?

Reply:
Point in PostGIS comes under GEOMETRY. Example : `GEOMETRY('POINT(1 2)')`, like `GEOMETRY('LINESTRING(1 2, 3 4)')`
The point data type I'm adding support for in this PR is Postgres' point type